test/corpus/object/objectMember.txt

```diff
@@ -13,10 +13,10 @@ res4 {1;2;3;}
 res5{1}
 res6{1;2}
 res7{foo = 1}
-res8{foo = 1 bar = 2}
+res8{foo = 1; bar = 2}
 res9{nested{1}}
-res10{["foo"] = 1 ["bar"] = 2}
-res11 { new {} new {} new {} }
+res10{["foo"] = 1; ["bar"] = 2}
+res11 { new {}; new {}; new {} }
```

Comment:
Keeping some negative (`ERROR`) cases in might be valuable too (considering allowing non-semi-colon-ed same-line object literals is explicitly undesirable)

Reply:
It's actually a bug that it's being parsed this way.

But: only pkl-core parses this correctly; both pkl-intellij and tree-sitter-pkl generate incorrect parse trees here.